k3os/overlay/libexec/k3os/boot

```diff
@@ -34,14 +24,27 @@ EOF
             ;;
         console=*)
             CONSOLE_SPEC=${x#console=}
-            IFS=, read TTY BAUDRATE _ <<<"${CONSOLE_SPEC},9600"
+            IFS=, read TTY BAUDRATE _ <<<"${CONSOLE_SPEC},38400"
```

Comment:
9600 is the kernel default, any reason we want to change it?

Reply:
The intention is to make tty1 use baudrate `38400` like [the default in the k3os](https://github.com/rancher/k3os/blob/4738bb8f7c4bfcb22656e536077d71bd30c1138f/overlay/libexec/k3os/boot#L5).
Will fix! Thanks.

Reply:
The default is changed back to `9600`.